roles/openshift_logging/defaults/main.yml

```diff
@@ -3,7 +3,7 @@ openshift_logging_image_prefix: docker.io/openshift/origin-
 openshift_logging_image_version: latest
 openshift_logging_use_ops: False
 master_url: "https://kubernetes.default.svc.{{ openshift.common.dns_domain }}"
-public_master_url: "https://{{openshift.common.public_hostname}}:8443"
+openshift_logging_master_public_url: "https://{{openshift.common.public_hostname}}:8443"
```

Comment:
I was wondering about this value along with 'master_url'.  I would expect users to never change these values unless they are deploing outside of the cluster.  That scenario seems like it would be unsupported and users are left to their own devices to make it work.  Regarding master_url, is there any reason we should not make it 'private' and put it int vars?